node/examples/ai-sdk-openai-demo.mjs

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
```

Comment:
Is this really necessary? I don't think we are running this as an executable, right?